explanation/jaas_authorisation.rst

```diff
@@ -0,0 +1,30 @@
+JAAS Authorisation
+==================
+
+JAAS provides enterprise-level features on top of Juju. One such features is `Relationship-Based Access Control (ReBAC) <https://en.wikipedia.org/wiki/Relationship-based_access_control>`_ that enables enterprises to have more control over users and their permissions to access underlying Juju resources (e.g. controllers or models). JAAS uses `OpenFGA <https://openfga.dev/>`_ as the back-end service for ReBAC. You can read more about Juju's permission model in `here <https://juju.is/docs/juju/user-permissions>`_.
```

Comment:
nitpick: rebac doesn't exactly give us control over users..

Reply:
How about this?

> JAAS provides enterprise-level features on top of Juju. One such features is enhanced authorisation, which provides enterprises with more control over user permissions to access underlying Juju resources (e.g. controllers or models). JAAS utilizes `Relationship-Based Access Control (ReBAC) <https://en.wikipedia.org/wiki/Relationship-based_access_control>`_ by using `OpenFGA <https://openfga.dev/>`_ as the back-end service for ReBAC. You can read more about Juju's permission model in `here <https://juju.is/docs/juju/user-permissions>`_.